.github/workflows/automatic-cherry-pick.yaml

```diff
@@ -0,0 +1,246 @@
+name: cherry-pick
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  process_comment:
+    # This job will parse the cherry-pick command and perform several sanity checks.
+    # Ideally we want all pull requests opened from a given command to be correct and successfull, so that
+    # developers don't have to reissue a cherry-pick command for a subset of the commits/branches.
+    # Therefore, we do all possible sanity checks in this job, before any pull request is opened in subsequent jobs.
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, 'cherry-pick')
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    outputs:
+      commits: ${{ steps.command.outputs.commits }}
+      branches: ${{ steps.command.outputs.branches }}
+      branch_matrix: ${{ steps.command.outputs.branch_matrix }}
+      status: ${{ steps.report.outputs.status }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          # We need the history of all branches for the sanity checks
+          fetch-depth: 0
+
+      - name: Parse Command
+        # TODO: make this step into a separate job, so that we can handle more than one command per comment
+        id: command
+        shell: bash
+        run: |
+          set -f
+          command_string=$(echo "${{ github.event.comment.body }}" | grep -m 1 cherry-pick)
```

Comment:
I know it is in the pull request, but it might also be helpful to have an example or syntax of this `cherry-pick` command in this workflow file, so future contributors can understand how this step parses the command. Adding a `cherry-pick <hash_1> <hash_2> ... <hash_n> into <branch_1> <branch_2> ... <branch_n>` comment to the step would be great :)

Reply:
Done.